SmartCarMQTTApp/app/src/main/java/com/example/smartcarmqttapp/screens/quiz/QuizQuestionActivity.java

```diff
@@ -125,32 +131,15 @@ protected void onCreate(Bundle savedInstanceState) {
         bottomNavigationView.setSelectedItemId(R.id.practiceTheory);
 
         //add questions to question list via helper method --> help us select question
-        db = new CrushersDataBase(this);
+        CrushersDataBase db = new CrushersDataBase(this);
         questionList = db.getAllQuestions();
-        Collections.shuffle(questionList);
```

Comment:
Why was this removed, the intention was to spice up the questions when a user restarts a quiz; otherwise they will simply just keep going through the same sequence of questions

Reply:
Everyday im shuffling XD

Reply:
> Why was this removed, the intention was to spice up the questions when a user restarts a quiz; otherwise they will simply just keep going through the same sequence of questions

Good catch. It was throwing NPE and I must've accidentally deleted it instead of commenting out